src/main/java/com/google/sps/servlets/Post.java

```diff
@@ -22,23 +22,23 @@
 import javax.servlet.http.HttpServletRequest;
 
 public class Post {
-    String postId = "";
-    String organizationName = "";
-    int month = 0;
-    int day = 0;
-    int year = 0;
-    int startHour = 0;
-    int startMinute = 0;
-    int endHour = 0;
-    int endMinute = 0;
-    String location = "";
-    Double lat = 0.0;
-    Double lng = 0.0;
-    String numberOfPeopleItFeeds = "";
-    String typeOfFood = "";
-    String description = "";
-    String collegeId = "";
-    int timeSort = 0;
+    public String postId = "";
+    public String organizationName = "";
+    public int month = 0;
+    public int day = 0;
+    public int year = 0;
+    public int startHour = 0;
+    public int startMinute = 0;
+    public int endHour = 0;
+    public int endMinute = 0;
+    public String location = "";
+    public Double lat = 0.0;
+    public Double lng = 0.0;
+    public String numberOfPeopleItFeeds = "";
+    public String typeOfFood = "";
+    public String description = "";
+    public String collegeId = "";
+    public int timeSort = 0;
```

Comment:
What's the reason for doing this?

Reply:
It might be better practice to make getters instead

Reply:
Good style will dictate that this should be (1) made final (not possible in this case) or (2) just use getters (it's a lot and it's annoying) or use (3) an Immutable object builder (possibly the best option).  

Let me know what you decide on add some justification here.

Reply:
In order to get information from the Post to put into the email, I made all the variables public but not static or final because they change and are different for each instance. For getters, it would be a lot of functions to create. We ended up going with the Immutable object builder because it would make the Post immutable and we could then make all variables `public final` (which will be good style!).